src/editor/components/styles/controls.scss

```diff
@@ -8,6 +8,7 @@
   background: none;
   border-radius: $border-radius;
   margin: spacing(1) / 2;
+  padding: 6px;
```

Comment:
Might just be good to leave a comment here for why we've set it to 6px specifically.

Reply:
Until we pass a UN resolution making Chrome the only legal browser for consumer use, I've added a comment explaining things: https://github.com/UWmathNEWS/slugline-web/pull/68/commits/2af1588adeba9c877b4d77a68183da0275b26c85